src/test/fake/fakeTelemetryService.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TelemetryPublisher } from '../../shared/telemetry/telemetryPublisher'
+import { TelemetryFeedback } from '../../shared/telemetry/telemetryFeedback'
+
+export class FakeTelemetryPublisher implements TelemetryPublisher {
```

Comment:
I hope we will put everything in a single folder, `fake/`. I don't see why we should search around dozens of subfolders just to find fakes, or test utils.

Actually, does anyone ever find the subfolder hierarchy useful at all? I bet it wouldn't matter if we had the entire project as a flat list of files in a single folder, it would be just as confusing as it is now (probably less).